enterprise/frontend/src/metabase-enterprise/settings/selectors.ts

```diff
@@ -55,7 +55,7 @@ export function getLoginPageIllustration(
         isDefault: true,
       };
 
-    case "no-illustration":
+    case "none":
```

Comment:
adjust the value to match the value in [analytics](https://www.notion.so/metabase/Let-white-labeling-customers-customize-illustrations-24e120498d5f414aaf52ddd0de9e20fc?pvs=4#0f9388c8dac74020970f29f485f80989) for simplicity